utils/raspberrypi/readme.md

```diff
@@ -0,0 +1,26 @@
+# Utilities for Raspberry Pi
+
+This directory contains different utilities may be needed on the Raspberry Pi.
```

Comment:
Shouldn't it be "utilities that/which may be needed"?

Reply:
yes, thanks